includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4412,19 +4412,19 @@ private function validate_field( string $field_key, $field_value ) {
-	private function get_checkout_session_from_order( WC_Order $order ): ?object {
+	private function get_checkout_session_from_order( WC_Order $order ): ?WC_Stripe_Checkout_Session {
 		$checkout_session_id = WC_Stripe_Order_Helper::get_instance()->get_stripe_checkout_session_id( $order );
 		if ( ! $checkout_session_id ) {
 			return null;
 		}
 
-		$cache_key        = 'checkout_session_' . $checkout_session_id;
-		$checkout_session = WC_Stripe_Database_Cache::get( $cache_key );
-		if ( ! $checkout_session ) {
+		$cache_key   = 'checkout_session_' . $checkout_session_id;
+		$raw_session = WC_Stripe_Database_Cache::get( $cache_key );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Database_Cache' in method 'get_checkout_session_from_order'. (undefined)

(StaticAccess)